open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/FVTConstants.java

```diff
@@ -8,7 +8,7 @@
 public class FVTConstants
 {
     public static final String USERID = "erinoverview";
-    public static final String DEFAULT_URL = "https://localhost:9443";
+    public static final String DEFAULT_URL = "https://localhost:9444";
```

Comment:
Why are we changing the port here as it then does not match the ports we set up in postman collections which set up  for cocoMDS1 and 4?

Reply:
* I agree with your point about consistency
* In our samples/tests we tend not to document the 'required' configuration to run them. The best we tend to have (IMO) is the 'coco pharma' environment which a) is coherent and b) I know is being used by others even to test clients against and c) is how I test much of the release currently .The master template I used for this was generally the coco pharma notebook diagrams.  I also accept that a test is quite different to a tutorial - the requirement is more just 'I can run it'.
* In the notebooks, both cocoMDS1 and cocoMDS4 are on the 'data lake' omag server platform. Under 'common' the default URL for the data lake platform is defined as `dataLakePlatformURL  = os.environ.get('dataLakePlatformURL', 'https://localhost:9444')` - hence why the default setting.
* So this means at least if running a comparable coco setup locally, the FVT's will 'just work' - a placeholder for further automation
* From a search I couldn't find explicit use of the coco servers/platforms in the postman collections/environment for the dojo. Geared towards a developer role it's a more manual approach. The reason for changing the test above is if it looks like coco, and smells like coco, it is coco ;-)
* In #426 I've been looking at options to automate - I plan to create a new PR (on 2.1+) for that, which will require changing the parm handling here to accept input parms (in addition) - for a single server environment, which is IMO quite different to coco pharma. Along with this we'll add docs, and this would superceded any change similar to this